packages/kmod-5.15-nvidia/disable-gsp-tmpfiles.conf

```diff
@@ -0,0 +1 @@
+d /etc/modprobe.d 0750 root root -
```

Comment:
This should either go in `release.spec`, or we should change the unit to `mkdir /etc/modprobe.d` instead. Otherwise it's weird for a kmod package to own creating a system directory.

Reply:
Changed to use systemd-tmpfiles.